modules/admin_manual/pages/maintenance/manual_upgrade.adoc

```diff
@@ -1,142 +1,109 @@
 = Manual ownCloud Upgrade
 :toc: right
+:experimental:
 
 == Introduction
 
 This section describes how to manually upgrade your ownCloud installation.
 
-If you’re not comfortable performing a manual upgrade, you can also use your Linux distribution’s
-xref:installation/linux_installation.adoc[package manager].
-
-[[backup-your-existing-installation]]
-== Backup Your Existing Installation
+[[enable-maintenance-mode]]
```

Comment:
In this case, an explicit anchor isn't required, as one exactly the same will be generated, based on the title below.